ConsoleApp/Program.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using System.Runtime;
+using System.IO;
+using System.Text;
+using static System.Console;
+const string filename = "data.csv";
+
+void ValidateIsNumber(string value)
+{
+    try
+    {
+        Int32.Parse(value);
```

Comment:
what if your number includes '+', '(' or '-' e.g. `+38(050)123-45-67`?
here is a better way to validate the phone if you want to :)
https://www.abstractapi.com/guides/c-validate-phone-number